### PR TITLE
goerliv2 testnet deploy

### PR DESCRIPTION
**Pull request by 0x0aa0**
## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: contracts/script/deploy/goerliv2/output/GV2_prod_deployment_data.json
Comment (jianoaix) [inline contracts/script/deploy/goerliv2/output/GV2_prod_deployment_data.json:1]: "GV2" can be dropped, since it's already under the "/goerliv2" directory.

Comment (0x0aa0) [reply contracts/script/deploy/goerliv2/output/GV2_prod_deployment_data.json:1]: I would like to keep the naming as these deployment files are often used across repos to read in addresses for the deployment

**Review by ian-shim — COMMENTED** [review]
I see several updates to EL contracts. Are they any significant updates in any of them? Do we need to update the bindings?

**Comment by 0x0aa0** [comment]
> I see several updates to EL contracts. Are they any significant updates in any of them? Do we need to update the bindings?

afaik the only major interface change is an additional view function on the delegation manager

**Comment by ian-shim** [comment]
> afaik the only major interface change is an additional view function on the delegation manager

Submodule should be updated with the updated bindings. Can we have a separate PR that updates submodule + bindings and this one with just updates to deploy script?

**Review by ian-shim — APPROVED** [review]

**Pull request merged**